server/cpho/jinja2/indicators/upload_indicator.jinja2

```diff
@@ -146,17 +150,61 @@
                 <td>{{ tdt("Open text field YYYY-YYYY") }}</td>
               </tr>
               <tr>
-                <td>{{ tdt("Value_Displayed") }}</td>
+                <td>{{ tdt("Value_Unit") }}</td>
                 <td>{{ tdt("Value unit") }}</td>
+                <td>
+                  <ul>
+                    <li>{{ tdt("Age-Standardized Rate") }}</li>
+                    <li>{{ tdt("Crude Rate") }}</li>
+                    <li>{{ tdt("Defined Daily Dose/1,000 Census") }}</li>
+                    <li>{{ tdt("Percentage") }}</li>
+                    <li>{{ tdt("Percentage (Crude Rate)") }}</li>
+                    <li>{{ tdt("Rate per 10,000 Patient Days") }}</li>
+                    <li>{{ tdt("Rate per 100,000") }}</li>
+                    <li>{{ tdt("Rate per 100,000 (Crude Rate)") }}</li>
+                    <li>{{ tdt("Rate per 100,000 Live Births") }}</li>
+                    <li>{{ tdt("Years") }}</li>
+                    <li>{{ tdt("Other") }}</li>
+                  </ul>
+                </td>
+              </tr>
+              <tr>
+                <td>{{ tdt("Value_Displayed") }}</td>
+                <td>{{ tdt("Value Displayed") }}</td>
                 <td>
                   <ul>
                     <li>{{ tdt("%") }}</li>
-                    <li>{{ tdt("PER 100,000") }}</li>
-                    <li>{{ tdt("YEARS") }}</li>
-                    <li>{{ tdt("PER 1,000 CENSUS INHABITANTS") }}</li>
-                    <li>{{ tdt("PER 10,000 PATIENT DAYS") }}</li>
-                    <li>{{ tdt("PER 100,000 LIVE BIRTHS") }}</li>
-                    <li>{{ tdt("OTHER") }}</li>
+                    <li>{{ tdt("Per 1,000 census inhabitants") }}</li>
+                    <li>{{ tdt("Per 10,000 patient days") }}</li>
+                    <li>{{ tdt("Per 100,000") }}</li>
+                    <li>{{ tdt("Per 100,000 live births") }}</li>
+                    <li>{{ tdt("Years") }}</li>
+                    <li>{{ tdt("Other") }}</li>
+                  </ul>
+                </td>
+              </tr>
+              <tr>
+                <td>{{ tdt("Pt_Data_Availability") }}</td>
+                <td>{{ tdt("Pt Data Avalability") }}</td>
+                <td>
+                  <ul>
+                    <li>{{ tdt("%") }}</li>
+                    <li>{{ tdt("Available") }}</li>
+                    <li>{{ tdt("Suppressed") }}</li>
+                    <li>{{ tdt("Not available") }}</li>
+                  </ul>
+                </td>
+              </tr>
+              <tr>
+                <td>{{ tdt("Data_Quality") }}</td>
+                <td>{{ tdt("Data Quality") }}</td>
+                <td>
+                  <ul>
+                    <li>{{ tdt("Caution") }}</li>
+                    <li>{{ tdt("Acceptable") }}</li>
+                    <li>{{ tdt("Good") }}</li>
+                    <li>{{ tdt("Suppressed") }}</li>
+                    <li>{{ tdt("Very Good")}}</li>
```

Comment:
load these in from the upload mapper
use the living arrangement PR for reference
https://github.com/PHACDataHub/cpho-phase2/pull/113/files#diff-c484d9604ef50ed4ea47591ff3d938b58d9ba2d25d32396bdb0f49e647f714ba